lighthouserc.json

```diff
@@ -9,8 +9,7 @@
       }
     },
     "collect": {
-      "staticDistDir": "./public",
-      "url": ["http://localhost/", "http://localhost/adafruit/index.html"]
+      "staticDistDir": "./public/adafruit"
```

Comment:
Is this intentional?

Reply:
If the script works, this file will be removed. Left the file for now while I am testing.